docs/usertutorial.rst

```diff
@@ -0,0 +1,455 @@
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Exploring data with Apache Superset
+===================================
+
+In this tutorial, we will introduce key concepts in Apache Superset through
+the exploration of a real dataset which contains the flights made by employees
+of a UK-based organization in 2011. The following information about each
+flight is given:
+
+- The traveller's department. For the purposes of this tutorial
+  the departments have been renamed Orange, Yellow and Purple.
+- The cost of the ticket.
+- The travel class (Economy, Premium Economy, Business and First Class).
+- Whether the ticket was a single or return.
+- The date of travel.
+- Information about the origin and destination.
+- The distance between the origin and destination, in kilometers (km).
+
+Obtaining and loading the data
+------------------------------
+
+Download the data for this tutorial to your computer from `Github
+<https://raw.githubusercontent.com/apache-superset/examples-data/master/tutorial_flights.csv>`_.
+
+In the top menu, select **Sources > Upload a CSV**.
+
+.. image:: images/usertutorial/upload_a_csv.png
+
+Then, enter the **Table name** as tutorial_flights and select the **CSV file**
+from your computer.
+
+.. image:: images/usertutorial/csv_to_database_configuration.png
+
+Next enter the text Travel Date into the **Parse Dates** field.
+
+.. image:: images/usertutorial/parse_dates_column.png
+
+Leaving all the other options in their default settings, select **Save** at
+the bottom of the page.
+
+
+Table Visualization
+-------------------
+
+In this section, we’ll create our first visualization: a table to show the
+number of flights and cost per travel class.
+
+To create a new chart, select the **New** **>** **Chart**.
+
+.. image:: images/usertutorial/add_new_chart.png
+
+Once in the **Create a new chart** dialogue, select tutorial_flights
+from the **Chose a datasource** dropdown.
+
+.. image:: images/usertutorial/add_new_chart.png
+
+Next, select the visualization type as Table.
+
+.. image:: images/usertutorial/chose_a_datasource.png
+
+Then, select **Create new chart** to go into the chart view.
+
+By default, Apache Superset only shows the last week of data: in our example,
+we want to look at all the data in the dataset. No problem - within the
+**Time** section, remove the filter on **Time range** by selecting on
+**Last week** **> No filter**, with a final **OK** to confirm your
+selection.
+
+.. image:: images/usertutorial/no_filter_on_time_filter.png
+
+Now, we want to specify the rows in our table by using the **Group by**
+option. Since in this example, we want to understand different Travel
+Classes, we select Travel Class in this menu.
```

Comment:
nit: replace Travel Class by `Travel Class`

Reply:
I've done this using :guilabel: see [here for more info](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html)

![image](https://user-images.githubusercontent.com/45243854/68357239-a6d7f080-00e2-11ea-8cf1-e2c7f9d82992.png)